src/frontend/citizen-portal/nginx.conf

```diff
@@ -19,4 +19,4 @@ server {
     root   /usr/share/nginx/html;
   }
 
-}
\ No newline at end of file
+# }
```

Comment:
```suggestion
}

```

Reply:
Removed commented sections of the nginx.conf, should resolve this.